File: Modul-2/Soal-4/PRAK204-2410817210027-HafizPerdana.py
inputs = []
while True:
    line = input()
    if not line:
        break
    inputs.extend(line.split())

inputs = list(map(float, inputs))

jarijari, tinggi = inputs

volume = 3.14 * jarijari * jarijari * tinggi
luas = 2 * 3.14 * jarijari * (jarijari + tinggi)
keliling = 2 * 3.14 * jarijari

print(f"Volume = {volume:.2f}")
print(f"Luas = {luas:.2f}")
print(f"Keliling = {keliling:.2f}")

File: Modul-2/Soal-4/PRAK204-2410817210027-HafizPerdana.c
#include <stdio.h>

int main() {
    float jarijari, tinggi, volume, luas, keliling;

    printf("");
    scanf("%f", &jarijari);

    printf("");
    scanf("%f", &tinggi);

    volume = 22 * jarijari * jarijari * tinggi / 7;

    luas = 2 * 22 * jarijari * (jarijari + tinggi) / 7;

    keliling = 2 * 22  * jarijari / 7;

    printf("\n");
    printf("Volume = %.2f\n", volume);
    printf("Luas = %.2f\n", luas);
    printf("Keliling = %.2f\n", keliling);

    return 0;
}